### PR TITLE
Closes #47 streamline admiralpeds package calls

### LINKED ISSUE #47: Streamline admiralpeds-package.R
@zdz2101  We took our starter copy of https://github.com/pharmaverse/admiralpeds/blob/main/R/admiralpeds-package.R from admiraltemplate but it contains lots of imported functions that we don't actually use, so once your 2 new functions are merged can you reduce this down to only what's needed please? e.g. see admiralonco version for reference: https://github.com/pharmaverse/admiralonco/blob/main/R/admiralonco-package.R

Also if you spot any packages we don't need at all then please update DESCRIPTION file too.

### PR DESCRIPTION
**Pull request by zdz2101**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiralpeds (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiralpeds site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by zdz2101** [comment]
@rossfarrugia This should be ready the second the two functions are merged in too if you wanna take an early look

**Comment by rossfarrugia** [comment]
@zdz2101 i merged in the 2 functions now but there's some R CMD check issues - please can you take a look and resolve and then happy to re-review

**Comment by zdz2101** [comment]
> @zdz2101 i merged in the 2 functions now but there's some R CMD check issues - please can you take a look and resolve and then happy to re-review

Thanks, looks like I was missing just a couple more things, should be good now

File: DESCRIPTION [hunk 1]
```diff
@@ -19,27 +19,15 @@ URL: https://pharmaverse.github.io/admiralpeds,
 Depends: 
     R (>= 4.1)
 Imports:
-    admiral (>= 1.0.0),
```
Comment (rossfarrugia) [inline DESCRIPTION:22]: @zdz2101 i would expect this file to have closer matching imports/suggests lists to admiralonco (except maybe few less as we probably don't use as much as that one): https://github.com/pharmaverse/admiralonco/blob/main/DESCRIPTION, rather than at the moment we seem to have way less here than admiralonco and the other admiral pkg exts. like why wouldn't we have admiral under imports? and do we need to do >=1.1.0 for admiral and admiraldev when >=1.0.0 is probably fine here

Comment (zdz2101) [reply DESCRIPTION:22]: In terms of the most stripped down-version of a DESCRIPTION page for our package I believe this is it, we don't actually use any `admiral` functions except for the examples and template so it isn't a hard dependency/needed per se for imports, I can move it over there for the sake of it/looks, and can certainly modify to `>=1.0.0` too, is that preferred?

Comment (zdz2101) [reply DESCRIPTION:22]: image


Actually it creates this note if I move admiral to the depends

Comment (rossfarrugia) [reply DESCRIPTION:22]: wow, ok! surprised we didn't use any admiral functions to help build our 2 functions here. no problem though. i guess just make admiraldev (>=1.0.0) then and we should be good to go with this one.

**Review by rossfarrugia — APPROVED** [review]
